css/custom.css

```
@@ -668,6 +668,31 @@ body.modal-open .modalcontent {
   margin-right: 10%;
 }
 
+.quiz-right-bottom table {
+  width: 90%;
+  justify-content: center;
+  margin-bottom: 5%;
+  font-family: 'Press Start 2P';
+  font-size: 16px;
+  text-align: center;
+  color: white;
+  border-style: hidden;
+}
+
+.quiz-right-bottom th {
+  border: none;
+  color: orange;
+  text-align: center;
+}
+
+.quiz-right-bottom td {
+    border-bottom: solid .2px DimGray;
+    border-top: none;
+    border-left: none;
+    border-right: none;
```

```suggestion
    border: none;
    border-bottom: solid .2px DimGray;
```
think this should be simplified